flink-libraries/flink-table/src/test/scala/resources/testFilter1.out

```diff
@@ -47,8 +48,8 @@ Stage 3 : Data Source
 		Avg. Output Record Size (bytes) : (none)
 		Filter Factor : (none)
 
-		Stage 1 : Filter
-			content : ('a * 2) === 0
+		Stage 1 : FlatMap
+			content : where: (=(MOD(a, 2), 0)), select: (a, b)
```

Comment:
I chose `where` because it is more SQL-style and `where` is also an alias of `filter` in the Table API. We can change it, but IMO `where` is more consistent with the other operator names.